conda-forge.yml

```diff
@@ -10,3 +10,4 @@ test_on_native_only: true
 github:
   branch_name: main
   tooling_branch_name: main
+build_with_mambabuild: False
```

Comment:
Can be removed once https://github.com/conda-forge/conda-smithy/pull/1594 is merged and released.